gadget/install/partition_test.go

```diff
@@ -45,6 +47,15 @@ func (s *partitionTestSuite) SetUpTest(c *C) {
 	s.BaseTest.SetUpTest(c)
 
 	s.dir = c.MkDir()
+	s.gadgetRoot = filepath.Join(c.MkDir(), "gadget")
+
+	s.cmdPartx = testutil.MockCommand(c, "partx", "")
+	s.AddCleanup(s.cmdPartx.Restore)
+
+	cmdSfdisk := testutil.MockCommand(c, "sfdisk", `echo "sfdisk was not mocked"; exit 1`)
```

Comment:
ah this is really nice I like this, thanks for setting this up, we should do this for more things (mock a broken command in setup so tests cleanly fail if they forget to mock something)